src/hooks/useSend.ts

```diff
@@ -0,0 +1,43 @@
+import { useMutation, UseMutationReturnType } from 'wagmi/query'
```

Comment:
What advantages we have by exposing a useSend and useConfirm hooks over returning the send and confirm functions from the useSafe one?

Reply:
The `useSend` hook does not only provide a `send` function but the entire [useMutation](https://tanstack.com/query/v5/docs/framework/react/reference/useMutation) return object. That way you also get all the status properties (such as `isPending`, `isError` etc.) and can use them in your React component to update your view accordingly. For example something like this:

```ts
function Transaction() {
  const { send, isPending, isSuccess, data, isError, error } = useSend()

  return (
    <>
      <button onClick={() =>send({ transactions: [...] })}>Send</button>

      {isPending && <CircularProgress />}
      {isSuccess && <div>Transaction sent: {data.description}</div>}
      {isError && <div>Transaction failed: {error.message}</div>}
    </>
  )
}
```

Reply:
From my understanding it is the approach we had agreed on in the [sync meeting](https://www.notion.so/safe-global/Sync-over-research-options-31144853e89a484d828b1e3c2f9d6df9#9c4fc7b67c23476cb2d73fff0e96790d)
> - Use a generic hook for common requests
> - Use individual hook for those complex calls (sendTransaction, sendUserOperation, sendOffChainMessage…)

Reply:
I just realized that the hook might better be renamed to `useSendTransaction` (instead of `useSend`)

Reply:
Renamed the hooks